doc/tutorials/quickstart_in_rviz/test/bringup_test.cpp

```diff
@@ -0,0 +1,95 @@
+// Description: Test if launching the demo is successful
+
+#include <control_msgs/action/follow_joint_trajectory.hpp>
+#include <moveit_msgs/action/move_group.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+
+#include <gtest/gtest.h>
+#include <stdlib.h>
+
+namespace
+{
+// This is a bit of a hack to make thread sanitizer ignore a race condition
+// in the constructor of the rclcpp::Node
+#if defined(__has_feature)
+#if __has_feature(thread_sanitizer)
+__attribute__((no_sanitize("thread")))
+#endif
+#endif
+rclcpp::Node::SharedPtr
+make_node(std::string const& name, rclcpp::NodeOptions const& options)
+{
+  return std::make_shared<rclcpp::Node>(name, options);
+}
+}  // namespace
+
+namespace moveit2_tutorials::quickstart_in_rviz
+{
+class BringupTestFixture : public testing::Test
+{
+public:
+  BringupTestFixture()
+    : node_(std::make_shared<rclcpp::Node>("basic_bringup_test"))
+    , executor_(std::make_shared<rclcpp::executors::MultiThreadedExecutor>())
+  {
+  }
+
+  void SetUp() override
+  {
+    executor_->add_node(node_);
+    executor_thread_ = std::thread([this]() { this->executor_->spin(); });
+  }
+
+  void TearDown() override
+  {
+    executor_->cancel();
+    if (executor_thread_.joinable())
+    {
+      executor_thread_.join();
+    }
+  }
+
+protected:
+  rclcpp::Node::SharedPtr node_;
+  rclcpp::Executor::SharedPtr executor_;
+  std::thread executor_thread_;
+};
+
+TEST_F(BringupTestFixture, BasicBringupTest)
```

Comment:
Are there any other tests we could add here beyond bringup? Doesn't seem like it provides a ton of coverage